### PR TITLE
Doc'd that RawSQL can be used with __in.

### PR DESCRIPTION
**Pull request by simonw**
This is a very useful pattern which wasn't obvious from the documentation.

### CODE REVIEW

**Comment by simonw** [comment]
That test failure looks like a glitch to me: https://djangoci.com/job/pull-requests-windows/database=sqlite3,label=windows,python=Python39/10739/console

**Review by carltongibson — APPROVED** [review]
Hey @simonw — it's been a while 586a9dc4295357de1f5ad0590ad34bf2bc008f79 — welcome back! ⛵️

Looks fine. I've rebased to see if that CI issue disappears. (It happens... 🙂)

Thanks!

File: docs/ref/models/expressions.txt [hunk 1]
```diff
@@ -699,12 +699,16 @@ Sometimes database expressions can't easily express a complex ``WHERE`` clause.
...
-    >>> queryset.annotate(val=RawSQL("select col from sometable where othercol = %s", (someparam,)))
+    >>> queryset.annotate(val=RawSQL("select col from sometable where othercol = %s", (param,)))
 
 These extra lookups may not be portable to different database engines (because
 you're explicitly writing SQL code) and violate the DRY principle, so you
 should avoid them if possible.
 
+``RawSQL`` expressions can also be used as the target of ``__in`` filters::
+
+    >>> queryset.filter(id__in=RawSQL("select col from sometable where othercol = %s", (param,)))
```
Comment (pope1ni) [inline docs/ref/models/expressions.txt:710]: Just a small thing, but would it make sense to reference a column called `id` here?

```suggestion
    >>> queryset.filter(id__in=RawSQL("select id from sometable where col = %s", (param,)))
```

Comment (carltongibson) [reply docs/ref/models/expressions.txt]: OK, let's have that. Thanks @pope1ni

**Pull request merged**